shared/chat/conversation/index.native.js

```diff
@@ -55,25 +56,27 @@ const Conversation = (props: Props) => (
               showSearchSuggestions={props.showSearchSuggestions}
               style={{flex: 1}}
             />
-          : <Box style={{...globalStyles.flexBoxColumn, flex: 1}}>
-              <List
-                focusInputCounter={props.focusInputCounter}
-                listScrollDownCounter={props.listScrollDownCounter}
-                onEditLastMessage={props.onEditLastMessage}
-                onScrollDown={props.onScrollDown}
-                onFocusInput={props.onFocusInput}
-                editLastMessageCounter={props.editLastMessageCounter}
-              />
-              <Banner />
-              {props.showLoader && <LoadingLine />}
-              {props.finalizeInfo
-                ? <OldProfileResetNotice />
-                : <Input
+          : props.conversationIsError
```

Comment:
same here. this existing code should also change but we can do that in a different ticket